.circleci/config.yml

```diff
@@ -82,6 +82,14 @@ jobs:
        - run: ci/coverage_publish.sh
        - store_artifacts:
            path: /build/envoy/generated/coverage
+
+   clang-tidy:
```

Comment:
Nit: perhaps use `clang_tidy` to match style in rest of the file? (just pointing out, I'm fine with either, really)

Reply:
Also, you need to add this target to the workflow at the end of the file, otherwise it won't run.

Reply:
done